docs/docs/how-tos/nebari-local.md

```diff
@@ -159,6 +159,14 @@ security:
         tag: sha-b4a2d1e
 ```
 
+### Disable DNS for local deployment
+
+If you are deploying locally, DNS is not necessary except for dask dashboard and the notebook
```

Comment:
I would move this as a `:::note` under `## Initializing Nebari` and add a optional for `### Setting up a DNS record for the domain` this last bit could also be moved to the DNS section as well
> If you prefer to setup DNS, see the [domain-registry documentation](https://www.nebari.dev/docs/how-tos/domain-registry#what-is-a-dns).